src/renderer/tvgShape.cpp

```diff
@@ -130,11 +130,11 @@ Result Shape::appendCircle(float cx, float cy, float rx, float ry) noexcept
     auto ryKappa = ry * PATH_KAPPA;
 
     pImpl->grow(6, 13);
-    pImpl->moveTo(cx, cy - ry);
-    pImpl->cubicTo(cx + rxKappa, cy - ry, cx + rx, cy - ryKappa, cx + rx, cy);
+    pImpl->moveTo(cx + rx, cy);
```

Comment:
Changing it might introduce a regression bug, which we can address with another patch.

Reply:
I just wanted to comment this :)
@JSUYA - just so you know, it may cause some problems in tests in efl, as I remember correctly

Reply:
this change is moved to https://github.com/thorvg/thorvg/pull/1987